packages/sales-api-service/src/services/recurring-payments.service.js

```diff
@@ -0,0 +1,27 @@
+import { executeQuery, findDueRecurringPayments, RecurringPayment } from '@defra-fish/dynamics-lib'
+
+export const getRecurringPayments = date => executeQuery(findDueRecurringPayments(date))
+
+/**
+ * Process a recurring payment instruction
+ * @param transactionRecord
+ * @returns {Promise<{recurringPayment: null, contact: null}>}
```

Comment:
This is incorrect, there is never a contact key so you should omit it.
Also, recurringPayment should be (RecurringPayment | null), as it isn't always a null value